src/EnergyPlus/Construction.hh

```diff
@@ -216,32 +216,32 @@ namespace Construction {
         Real64 AbsDiffShade = 0.0;            // Diffuse solar absorptance for shade
         Real64 AbsDiffBackShade = 0.0;        // Diffuse back solar absorptance for shade
         Real64 ShadeAbsorpThermal = 0.0;      // Diffuse back thermal absorptance of shade
-        Array1D<Array1D<Real64>> AbsBeamCoef; // Coefficients of incidence-angle polynomial for solar
+        Array1D<std::array<Real64, DataSurfaces::MaxPolyCoeff>> AbsBeamCoef; // Coefficients of incidence-angle polynomial for solar
```

Comment:
Polynomial coefficients are now held in `std::array` because the maximum number is known at compile time.